upgrades/v5.0.4/README.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Neutron v5.0.4 Upgrade
+order: 2
+---
+
+<!-- markdown-link-check-disable -->
+
+# Neutron v5.0.4 Upgrade, Instructions
+
+- Chain upgrade point: `December 20th 2024, 14:00 UTC` approximately at height `18015000`;
+- Go version: `v1.22`
+- Release: https://github.com/neutron-org/neutron/releases/tag/v5.0.4
+
+This document describes the steps for validators and full node operators, to upgrade successfully to the Neutron v5.0.4 release. For more details on the release, please see the [release notes](https://github.com/neutron-org/neutron/releases/tag/v5.0.4).
+
+## Upgrade date
+
+The upgrade will take place approximately on December 20th at `14:00 UTC` approximately at height `18015000`;
+
+## Chain-id will remain the same
+
+The chain-id of the network will remain the same, `neutron-1`. This is because an in-place migration of state will take place, i.e., this upgrade does not export any state.
+
+### System requirement
+
+- 64GB RAM is recommended to ensure a smooth upgrade.
+
+If you have less than 64GB RAM, you might try creating a swapfile to swap an idle program onto the hard disk to free up memory. This can allow your machine to run the binary than it could run in RAM alone.
+
+```shell
+sudo fallocate -l 64G /swapfile
+sudo chmod 600 /swapfile
+sudo mkswap /swapfile
+sudo swapon /swapfile
+```
+
+- Make sure you have enough disk space for upgrade, the state can grow twice durng upgrade.
+
+### Backups
+
+Prior to the upgrade, validators are encouraged to take a full data snapshot. Snapshotting depends heavily on infrastructure, but generally this can be done by backing up the `.neutrond` directory.
+If you use Cosmovisor to upgrade, by default, Cosmovisor will backup your data upon upgrade. See below [upgrade using cosmovisor](#method-ii-upgrade-using-cosmovisor) section.
+
+It is critically important for validator operators to back-up the `.neutrond/data/priv_validator_state.json` file after stopping the neutrond process. This file is updated every block as your validator participates in consensus rounds. It is a critical file needed to prevent double-signing, in case the upgrade fails and the previous chain needs to be restarted.
+
+### Current runtime
+
+The Neutron mainnet network, `neutron-1`, is currently running [Neutron v5.0.2](https://github.com/neutron-org/neutron/releases/tag/v5.0.2). We anticipate that operators who are running on v5.0.2, will be able to upgrade successfully. Validators are expected to ensure that their systems are up-to-date and capable of performing the upgrade. This includes running the correct binary, or if building from source, building with go `1.22`.
```

Comment:
the current version seems to be 5.0.3: https://rpc-kralum.neutron-1.neutron.org/abci_info
```suggestion
The Neutron mainnet network, `neutron-1`, is currently running [Neutron v5.0.3](https://github.com/neutron-org/neutron/releases/tag/v5.0.3). We anticipate that operators who are running on v5.0.3, will be able to upgrade successfully. Validators are expected to ensure that their systems are up-to-date and capable of performing the upgrade. This includes running the correct binary, or if building from source, building with go `1.22`.
```

Reply:
No, it's only for rpc nodes, vals are running v5.0.2